File: provisioning/paging-only-groups/assign-paging-group-users-and-devices/code-samples/assignMultiplePagingGroupUsersDevices.php
<?php
// https://developers.ringcentral.com/my-account.html#/applications
// Find your credentials at the above url, set them as environment variables, or enter them below

// PATH PARAMETERS
$accountId = '<ENTER VALUE>';
$pagingOnlyGroupId = '<ENTER VALUE>';

// POST BODY
$body = array(
    'addedUserIds' => array(
        '<ENTER VALUE>'  
    ),
    'removedUserIds' => array(
        '<ENTER VALUE>'  
    ),
    'addedDeviceIds' => array(
        '<ENTER VALUE>'  
    ),
    'removedDeviceIds' => array(
        '<ENTER VALUE>'  
    )
);

require('vendor/autoload.php');
$rcsdk = new RingCentral\SDK\SDK(getenv('clientId'), getenv('clientSecret'), getenv('serverURL'));
$platform = $rcsdk->platform();
$platform->login(getenv('username'), getenv('extension'), getenv('password'));
$r = $platform->post("/restapi/v1.0/account/{$accountId}/paging-only-groups/{$pagingOnlyGroupId}/bulk-assign", $body);
// PROCESS RESPONSE
?>

File: provisioning/paging-only-groups/assign-paging-group-users-and-devices/code-samples/assignMultiplePagingGroupUsersDevices.py
# https://developers.ringcentral.com/my-account.html#/applications
# Find your credentials at the above url, set them as environment variables, or enter them below

# PATH PARAMETERS
accountId = '<ENTER VALUE>'
pagingOnlyGroupId = '<ENTER VALUE>'

# POST BODY
body = {
    'addedUserIds': [
        '<ENTER VALUE>'
    ],
    'removedUserIds': [
        '<ENTER VALUE>'
    ],
    'addedDeviceIds': [
        '<ENTER VALUE>'
    ],
    'removedDeviceIds': [
        '<ENTER VALUE>'
    ]
}

import os
from ringcentral import SDK
rcsdk = SDK(os.environ['clientId'], os.environ['clientSecret'], os.environ['serverURL'])
platform = rcsdk.platform()
platform.login(os.environ['username'], os.environ['extension'], os.environ['password'])
r = platform.post(f'/restapi/v1.0/account/{accountId}/paging-only-groups/{pagingOnlyGroupId}/bulk-assign', body)
# PROCESS RESPONSE